README.md

```diff
@@ -1,15 +1,42 @@
-# neuroid-reactnative-sdk
+# NeuroID React-Native Mobile SDK
 
-## Installation
+![Neuro-ID Logo](/docs/images/NeuroID_Logo.png)
 
-Visit https://neuro-id.readme.io/docs/react-native-installation for full installation documentation
+## Overview
 
-## Contributing
+NeuroID's React-Native Mobile SDK makes it simple to embed behavioral analytics inside your mobile application. With a few lines of code, you can connect your app with the NeuroID platform and make informed decisions about your users.
+
+## Installation and Requirements
+
+For installation instructions and minimum project requirements, please read our [React-Native Installation Guide](https://neuro-id.readme.io/docs/react-native-installation).
+
+## Usage
+
+To use the NeuroID React-Native Mobile SDK there are 4 basic steps:
+
+1. [Installation](https://neuro-id.readme.io/docs/react-native-installation#installation)
+1. [Configuration/Initialization of the SDK](https://neuro-id.readme.io/docs/react-native-installation#2-initialize-the-sdk)
+1. [Linking People to Data](https://neuro-id.readme.io/docs/react-native-installation#3-link-people-to-data)
+1. [Calling `RegisterPageTargets`](https://neuro-id.readme.io/docs/react-native-installation#4-calling-registerpagetargets)
+
+For full instructions, available API commands, and documentation please visit our [React-Native Installation Guide](https://neuro-id.readme.io/docs/react-native-installation).
 
-See the [contributing guide](CONTRIBUTING.md) to learn how to contribute to the repository and the development workflow.
+**Note:** Bad cache? Try `npx react-native start --reset-cache`
+
+## Examples
+
+To view examples of how to configure and use the NeuroID React-Native Mobile SDK in a real project please visit the following examples:
+
+1. [React-Native Example](https://neuro-id.readme.io/recipes/react-native-installation)
+
+## Support
+
+For support or troubleshooting of the NeuroID React-Native Mobile SDK please contact a NeuroID Customer Support Representative.
+
+## Contributing
 
-Bad cache? `npx react-native start --reset-cache`
+For a feature request in the NeuroID React-Native Mobile SDK please contact a NeuroID Customer Support Representative.
 
 ## License
 
-MIT
+The NeuroID React-Native Mobile SDK is provided under an [MIT License](LICENSE).
```

Comment:
No idea if this is worth calling out, but the other SDK's / Javascript dont have a license mentioned in the readme that I saw.